tests/unit/test_orderbook_rewards.py

```diff
@@ -1,34 +1,57 @@
 import unittest
 
 from src.models import AccountingPeriod
-from src.update.orderbook_rewards import (
-    dune_repr,
-    rewards_df_to_dune_list,
-    orderbook_rewards_query,
-)
+from src.update.orderbook_rewards import RewardQuery
 import pandas as pd
 
 
 class TestCowRewardsUpsert(unittest.TestCase):
     def setUp(self) -> None:
-        self.example = pd.DataFrame(
+        self.aggregate_example = pd.DataFrame(
             {"receiver": ["0x1", "0x2"], "amount": [1234, 5678], "num_trades": [46, 2]}
         )
-        self.expected = [
+        self.agg_expected = [
             "('0x1', 46, 1234)",
             "('0x2', 2, 5678)",
         ]
+        self.per_order_example = pd.DataFrame(
+            {
+                "solver": ["s1", "s2", "s3"],
+                "tx_hash": ["t1", "t2", "t3"],
+                "amount": [1234, 5678, 91011],
+                "safe_liquidity": [True, False, None],
+            }
+        )
+        # See https://dune.com/queries/1456810 for Null Test Demonstration
+        self.per_order_expected = [
+            "('s1', 't1', 1234, True)",
+            "('s2', 't2', 5678, False)",
+            "('s3', 't3', 91011, Null)",
+        ]
+        print(self.per_order_example)
+        print(self.aggregate_example)
 
     def test_dune_repr(self):
-        for i, (_, row) in enumerate(self.example.iterrows()):
-            self.assertEqual(self.expected[i], dune_repr(row))
+        for i, (_, row) in enumerate(self.aggregate_example.iterrows()):
+            self.assertEqual(self.agg_expected[i], RewardQuery.AGGREGATE.dune_repr(row))
+
+        for i, (_, row) in enumerate(self.per_order_example.iterrows()):
+            self.assertEqual(
+                self.per_order_expected[i], RewardQuery.PER_ORDER.dune_repr(row)
+            )
```

Comment:
```suggestion
        self.assertEqual(
            self.per_order_expected,
            list(
                self.per_order_example.apply(
                    lambda x: RewardQuery.PER_ORDER.dune_repr(x), axis=1
                )
            ),
        )
```

Reply:
I am going to commit your suggestions but I have a feeling they are going to fail... Maybe I don't understand what axis=1 means.